python/Ganga/GPIDev/Lib/Registry/BoxRegistry.py

```diff
@@ -155,14 +161,11 @@ def proxy_remove(self, obj_id):
 
         self._remove(self._get_obj(obj_id))
 
+    def getSlice(self):
+        return self.stored_slice
+
     def getProxy(self):
-        slice = BoxRegistrySlice(self.name)
-        slice.objects = self
-        proxy = BoxRegistrySliceProxy(slice)
-        proxy.add = self.proxy_add
-        proxy.rename = self.proxy_rename
-        proxy.remove = self.proxy_remove
-        return proxy
+        return self.stored_proxy
```

Comment:
We don't need 'getters' for these. It's more Pythonic to just access the names directly or at least to provide access to them via `@property`s